frontend/run-yarn-audit.sh

```diff
@@ -21,6 +21,16 @@ if [ -f yarn-audit-known-issues ] && echo "$output" | grep auditAdvisory | diff
 	exit 0
 fi
 
+
+curr=$(cat yarn-audit-known-issues | jq -s 'map({name: .data.advisory.module_name, version: .data.advisory.findings[0].version})| unique | sort_by(.data.advisory.module_name) | tostring')
+new=$(yarn audit --level low --json --groups dependencies | jq -s 'map(select(.type == "auditAdvisory")) | map({name: .data.advisory.module_name, version: .data.advisory.findings[0].version})| unique | sort_by(.data.advisory.module_name) | tostring')
+
+if [ "$curr" = "$new" ]; then
+    echo
+	echo Ignorning known vulnerabilities
```

Comment:
`Ignoring`

Also I think a little more explanation might be helpful here because the verbiage as it exists might be alarming to the uninitiated. Perhaps "Ignoring previously known and accepted vulnerabilities" or something similar. It also sort of serves as an explanatory comment for the script lines.